### PR TITLE
Add Leaflet example

### PR DESCRIPTION
**Pull request by ejke**
Add in files for Leaflet example - the final version and template to start the tutorial with.

### CODE REVIEW

File: Leaflet/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Leaflet API Integration App
+
+This project is intended to provide an easy-to-use example of how to integrate our Catalog & Image APIs into a Leaflet javascript application.
+
+To run the app:
+1. In file index.html, on line 95, add your Bird.i API token
```
Comment (austintino) [inline Leaflet/README.md:6]: Maybe add a link to the specific line?

**Review by austintino — APPROVED** [review]
Just a wee comment about the README but other than that, 👍 

🤘

File: Leaflet/README.md (hunk 1 above)
Comment (ejke) [reply Leaflet/README.md:6]: Sure thing - I just need to merge it before I can get the link!

**Review by sammccants — APPROVED** [review]
🔎

**Pull request merged**